### PR TITLE
[0.7] add astropy 1.0 to the 0.7 matrix

### PR DESCRIPTION
**Pull request by Cadair**
This PR is designed to make @nabobalis's life harder.

### CODE REVIEW

**Comment by nabobalis** [comment]
So are forgoing old numpy versions now?

Also should it not be 1.3?

**Comment by Cadair** [comment]
the minimum version for 0.7 is 1.0. And I just combined the old numpy and old astropy builds.

**Comment by nabobalis** [comment]
#Fairenough

File: .travis.yml [hunk 1]
```diff
@@ -33,9 +33,11 @@ env:
         - PYTHON_VERSION=2.7 SETUP_CMD='egg_info' 
         - PYTHON_VERSION=3.4 SETUP_CMD='egg_info'
```
Comment (nabobalis) [inline .travis.yml]: Lets drop this one

File: .travis.yml [hunk 2]
```diff
@@ -33,9 +33,11 @@ env:
...
         - PYTHON_VERSION=3.5 SETUP_CMD='egg_info'
+        - PYTHON_VERSION=3.6 SETUP_CMD='egg_info'
         - PYTHON_VERSION=2.7 SETUP_CMD='test'
         - PYTHON_VERSION=3.4 SETUP_CMD='test'
```
Comment (nabobalis) [inline .travis.yml]: And here.

File: .travis.yml (hunk 1 above)
Comment (Cadair) [inline .travis.yml]: no, we shipped with support for it, which means it stays.

Comment (nabobalis) [inline .travis.yml]: Oh this isnt master.

**Comment by nabobalis** [comment]
CI will break without the astropy change. So I will be merging this first.

**Pull request merged**